data/privacy.md

```diff
@@ -1,97 +1,73 @@
-- Honor opt-out/unsubscribe requests quickly.
-- Allow users to unsubscribe by using the link at the bottom of each email.
-
-If at any time you would like to unsubscribe from receiving future emails, you can email us at <allaboutolaf@stolaf.edu> and we will promptly remove you from ALL correspondence.
-
-## Google Analytics
-
-The app also uses Google Analytics to see how people use the app, which allow us to prioritize our development time more efficiently.
-
-## Contacting Us
-
-If there are any questions regarding this privacy policy, you may contact us using the information below.
-
-Student Government Assoc. ATTN: Webmaster
-1500 St. Olaf Ave.
-Northfield, MN 55057 USA
-<allaboutolaf@stolaf.edu>
-
-Last Edited on 2018-01-14
+We respect your right to privacy when using our app, All About Olaf.
+
+Our policy towards privacy is simple: we aim to collect no data other
+than what we need to keep the app functional and ensure a smooth
+experience for all users.
+
+## Who runs this app?
+
+This app is developed and maintained by a mixed team of current
+students at St. Olaf College and alumni of St. Olaf College.  Anyone
+is welcome to make contributions to the app, and the source code for
+the app is publicly available [on GitHub][repo].  All contributions go
+through code review by maintainers.
+
+This app is entirely self-funded, and we do not include advertisements
+on our app.  As such, our privacy policy is quite simple.
+
+[repo]: https://github.com/StoDevX/AAO-React-Native
+
+## What data do you collect?
+
+- When you "Sign in with St. Olaf" in the settings, your St. Olaf
+  username and password are stored locally, on your device, in the
+  encrypted keychain storage mechanism available on your device.  _We
+  do not transmit these credentials to or through any servers that we
+  control, and we do not have the ability to remotely access these
+  credentials_.  These credentials become inaccessible when you delete
+  the app.  We only transmit these credentials to on-campus servers,
+  and the security of those servers is not controlled by us.
+
+- When you fetch any data besides your personal information, your
+  requests may reach servers that we control.  Whenever your app makes
+  such a request, the following information is logged:
+  - _Your public IP address_, for diagnostic purposes.  We do not use
+    this to identify you or for tracking purposes, and this is logged
+    for diagnostic purposes.
+  - _The version of All About Olaf you are using and information about
+    your phone_, which is sent as part of your device's request.  This
+    data is intentionally vague so that we can also not use this to
+    identify you.
+  - _What you requested and how long it took_, which is widely
+    considered to be fully anonymous.
+
+- We also use a few tools to track our most used features and to
+  gather crash reports.  These tools are:
+  - _Google Analytics_, which can be disabled via the "Share
```

Comment:
@hawkrives raises a good point about the WiFi issue reporting. That’d be an important section to have.

Reply:
Oh, and something to think on. Do we want to say anything about OneSignal and notifications?

Reply:
1. Is it worth adding links to these tools?
2. Should we link to `ccc-server` so anyone can see its source?

Reply:
I addressed most of your suggestions in this thread, but:

1. I don't think we really need to—this would just be distracting to me.
1. I think the link to `StoDevX/AAO-React-Native` is sufficient; anyone who wants to dig deeper will probably realize rather quickly how this works.  Since it's referred to as "our server" I don't think it'll really need much more detail.

Feel free to mark this conversation as unresolved if you think this is a blocker.